src/components/overlays/Tooltip/TooltipProvider.tsx

```diff
@@ -224,7 +225,7 @@ export const TooltipProvider = (props: TooltipProviderProps) => {
   
   return (
     <>
-      {tooltipRendered}
+      {createPortal(tooltipRendered, document.body)}
```

Comment:
Please add a comment here explaining a portal is used here to prevent CSS rules from the parent from being inherited into the Tooltip.